code/modules/surgery/organs/organ_external.dm

```diff
@@ -552,6 +523,51 @@ Note that amputating the affected organ does in fact remove the infection from t
 			qdel(src) // If you flashed away to ashes, YOU FLASHED AWAY TO ASHES
 			return null
 
+/obj/item/organ/external/chest/droplimb()
+	if(cannot_amputate || !owner || !hasorgans(src))
+		return
+
+	var/mob/living/carbon/C = owner
+
+	var/organ_spilled = 0
+	var/turf/T = get_turf(C)
+	C.add_splatter_floor(T)
+	playsound(get_turf(C), 'sound/effects/splat.ogg', 25, 1)
+	for(var/X in C.internal_organs)
+		var/obj/item/organ/O = X
+		var/org_zone = check_zone(O.parent_organ)
+		if(org_zone == "chest" || org_zone == "groin")
```

Comment:
this only applies to the chest, so why is this checking for the zone of "groin"?

Reply:
Cuase otherwise it dosen't drop the organs in the groin...I maybe be able to do it a better way perhaps what do you suggest?